src/treeview.ts

```diff
@@ -112,8 +112,14 @@ export function startTreeView(
         if (i < params.uriChain.length) {
           const uri = params.uriChain[i];
           const isExpanded = expandedNode(params.viewId).has(uri);
+          const isDestinationNode = i == 0;
```

Comment:
=== for consistency (and safety)

Reply:
(I see you just moved the line, but it’s a good chance to fix it nonetheless)

Reply:
Fixed!